packages/terra-action-header/tests/jest/ActionHeader.test.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-extraneous-dependencies, no-alert */
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import Button from 'terra-button';
+import ActionHeader from '../../src/ActionHeader';
+import messages from '../../translations/en-US.json';
+
+const locale = 'en-US';
+
+describe('ActionHeader', () => {
+  // Snapshot Tests
+  it('should render a default action header', () => {
+    const actionHeader = shallow(
```

Comment:
Switch shallow statements to use `render` instead. It provides more meaningful information of the DOM.

Reply:
Updated [here](https://github.com/cerner/terra-core/commit/85b48f23579d7aebe02de8f192ff8110e6b2727e).